ixnetwork_open_traffic_generator/vport.py

```diff
@@ -615,8 +615,19 @@ def results(self, request):
             self._column_names = []
         else:
             self._column_names = request.column_names
+        
+        port_filter = {'property': 'name', 'regex': '.*'}
+        port_names = [port.name for port in self._api._config.ports]
+        if request is not None and request.port_names is not None \
+                and request.port_names:
+            port_names = request.port_names
+        if len(port_names) == 1:
+            port_filter['regex'] = '^%s$' % port_names[0]
+        elif len(port_names) > 1:
+            port_filter['regex'] = '^(%s)$' % '|'.join(port_names)
+            
```

Comment:
Shouldn't just executing the snippet under `else` (for regex) work for all cases ?

Reply:
we will ignore corner case when len(port_names) == 0. So I still vote for elif.